DQM/Integration/python/clients/beampixel_dqm_sourceclient-live_cfg.py

```diff
@@ -87,11 +87,10 @@
     process.load("RecoLocalTracker.Configuration.RecoLocalTracker_cff")
     process.load("TrackingTools.TransientTrack.TransientTrackBuilder_cfi")
     from RecoPixelVertexing.PixelLowPtUtilities.siPixelClusterShapeCache_cfi import *
-    process.siPixelClusterShapeCachePreSplitting = siPixelClusterShapeCache.clone(
-        src = 'siPixelClustersPreSplitting'
-    )
+    process.siPixelClusterShapeCachePreSplitting = siPixelClusterShapeCache.clone(src = cms.InputTag("siPixelClustersPreSplitting"))
```

Comment:
With @rovere we deliberately left the explicit type (`cms.InputTag` here) away, because this way the framework checks if the parameter (`src` here) already exists, and if not, gives an error. This is very useful e.g. against misspellings.